src/porepy/numerics/fem/rt0.py

```diff
@@ -363,4 +401,14 @@ def massHdiv(self, K, c_volume, coord, sign, dim, HB):
         return np.dot(C.T, np.dot(N.T, np.dot(HB, np.dot(inv_K, np.dot(N, C)))))
 
 
+# ------------------------------------------------------------------------------#
+
+    def opposite_side_node(self, face_nodes, nodes, faces_loc):
```

Comment:
1. Document function, I have no idea what it does.
2. Is this really much more general, and should be put somewhere related to grids?

Reply:
it relay on the fact that the grid is made of simplex, so not so general